src/loginServices.js

```diff
@@ -131,10 +131,18 @@ export const setUnauthorizedPathToSession = (pathname) => {
 };
 export const getUnauthorizedPathFromSession = () => sessionStorage.getItem(UNAUTHORIZED_PATH);
 
-const TENANT_LOCAL_STORAGE_KEY = 'tenant';
+export const getOIDCRedirectUri = (tenant, clientId) => {
+  // we need to use `encodeURIComponent` to separate `redirect_uri` URL parameters from the rest of URL parameters that `redirect_uri` itself is part of
+  return encodeURIComponent(`${window.location.protocol}//${window.location.host}/oidc-landing?tenant=${tenant}&client_id=${clientId}`);
```

Comment:
Petty: why rename `clientId` to `client_id` in the string you return? Just use `clientId` consistently, here and in `getStoredTenant()` too.

Reply:
Eh, I see other params to keycloak use snake_case, so `client_id` makes sense in that sense. Never mind; sorry for the noise.